### ISSUE TITLE
Debugging: Break on uncaught exceptions

### ISSUE THREAD

**Issue by aubade** [bruno-medeiros/DDT#93]
As it stands, the debugging support will let D's internal exception handler take any uncaught exceptions, which prevents you from getting the debugger's own backtrace. This can be worked around with a .gdbinit file containing "break _d_throwc", which triggers a breakpoint every time D throws an exception, handled or not, but it'd be really nice if DDT could hook up something to only break if the program doesn't catch an exception.

**Comment by bruno-medeiros** [comment]
You mean if the exception would go through the program's `main` and cause it to terminate?

**Comment by aubade** [comment]
Precisely, yes. Then rather than needing to read the backtrace from stdout, we could use the nice annotated backtrace that Eclipse gives on segfaults.
